edge/pkg/devicetwin/dtmanager/device_test.go

```diff
@@ -237,14 +244,13 @@ func TestDealDeviceStateUpdate(t *testing.T) {
 			updateReturnInt:  int64(1),
 			updateReturnErr:  nil,
 			queryTableReturn: querySeterMock,
-			times:            2,
```

Comment:
I try to fix this from 2 to 1, and ut passed.

Reply:
😲seem that you're right